### DISCUSSION TITLE
Cannot use PA0 pin since fan-control.cfg is locked

### DISCUSSION THREAD

**Discussion by gbrockelman** [Guilouz/Creality-Helper-Script-Wiki#576] [Issues]
Hi.
 
I recently added a Bento Box to my K1. I always forget to unplug the fan, so I want the printer to control the fans using a min/max limit for the chamber temperature. I found a guide on Printables explaining how to do this (link below). In the guide, they use the open PA0 pin on the motherboard, but I cannot adjust the fan-control.cfg since the Helper Script locks this file. Is there a way to bypass the lockout on the cfg file, or is there an alternative solution using a different file that the Helper Script does not lock?

https://www.printables.com/model/825056-bentobox-v20-5015-pull-vertical-orientation-for-cr

**Comment by michaelalanchase** [comment]
The 'files' in the Help-Script directory are just symbolic links elsewhere in the filesystem (to read only files), but they can be replaced with local copies in that directory (or moved to the same directory as printer.cfg) and edited as needed.
